app/javascript/controllers/map_controller.js

```diff
@@ -154,10 +154,10 @@ export default class extends Controller {
     const { zoomAmount } = this.imageTarget.dataset
 
 
-    const { x, y }  = this.mapPositionOf(event)
+    const { x, y } = this.mapPositionOf(event)
 
-    const newX = x + (parseFloat(offsetX) / parseFloat(zoomAmount))
-    const newY = y + (parseFloat(offsetY) / parseFloat(zoomAmount))
+    const newX = Math.floor(x + (parseFloat(offsetX) / parseFloat(zoomAmount)))
+    const newY = Math.floor(y + (parseFloat(offsetY) / parseFloat(zoomAmount)))
```

Comment:
I was getting intermittent failures on CI because this was coming out to a decimal number.